setup.cfg

```diff
@@ -10,4 +10,4 @@ lines_after_imports=2
 order_by_type=1
 known_first_party=weblab
 include_trailing_comma=True
-skip=wsgi.py,migrations,__init__.py,node_modules
```

Comment:
As an aside - I noticed that there is only a secrets.example.py file in the master not a secrets.py. I am assuming this is to force the installer to create their own copy.

Reply:
That's right. No committing sensitive information to a public repo :)
(`secrets.py` is also in the .gitignore file to help avoid people adding it by accident)